### ISSUE TITLE
[BUG] Accessibility: Disable Input labels and hint text fail WCAG 1.4.3

### ISSUE THREAD

**Issue by LukeTaylor34** [UKHO/admiralty-design-system#160]
**Describe the bug**
The Light grey Labels and hint text for Disabled inputs fail colour contrast WCAG 1.4.3.

**Expected behaviour**
The input label text needs to pass a contrast ratio of at least 4.5:1.

**Screenshots**
![image](https://github.com/UKHO/admiralty-design-system/assets/97240814/18fa28fb-7991-4347-9f44-cb468ad5b149)

**Additional context**
1.4.3 - Contrast - The visual presentation of text and images of text has a contrast ratio of at least 4.5:1, except for the following:
Large Text - Large-scale text and images of large-scale text have a contrast ratio of at least 3:1;
Incidental - Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.
Logotypes - Text that is part of a logo or brand name has no contrast requirement.

**Comment by LukeTaylor34** [comment]
After some further investigation this falls under - Incidental except - Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.

**Comment by KatiePUX** [comment]
Our hands are tied with this one due to marketing constraints, to be reviewed

**Comment by developernm** [comment]
#949494 & #ADADAD fail the accessibility tests
#757575 passes the tests for AAA

![Image](https://github.com/user-attachments/assets/c8be9b24-8dc8-46d3-887c-9012e424c5fd)

**Comment by developernm** [comment]
Closed [PR: fix: Make disabled colours pass AA colour contrast #205 ](#205) due to merge conflicts. 

Created new PR [Fix: Accessibility Disable Input labels and hint text fail WCAG 1.4.3](https://github.com/UKHO/admiralty-design-system/pull/288)
